sys/include/net/skald.h

```diff
@@ -54,19 +54,6 @@
 extern "C" {
 #endif
 
-/**
- * @defgroup net_skald_conf Skald compile configurations
- * @ingroup config
- * @{
- */
-/**
- * @brief   Advertising interval in microseconds
- */
-#ifndef CONFIG_SKALD_INTERVAL_MS
-#define CONFIG_SKALD_INTERVAL_MS        (1000U)
-#endif
```

Comment:
Does it make sense to keep this? Maybe not as a `CONFIG_` define, but at least as a default? This way you would not need to duplicate the defines it in the `main.c`s